src/interfaces/assistants_web/src/hooks/files.ts

```diff
@@ -140,11 +142,14 @@ export const useFileActions = () => {
       uploadedFiles.forEach((uploadedFile) => {
         newFileIds.push(uploadedFile.id);
         setParams({ fileIds: newFileIds });
-        addComposerFile(uploadedFile);
+        addComposerFile({ ...uploadedFile, user_id: userId });
       });
 
       if (!conversationId) {
-        setConversation({ id: uploadedFiles[0].conversation_id });
+        const newConversationId = uploadedFiles[0].conversation_id;
+        if (newConversationId) {
```

Comment:
probably good to wait for @scott-cohere changes to land before doing these changes

Reply:
@knajjars This was due to the new API client generation; since it's already on main and this is just typescript, the actual code already looks like this.